src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -50,7 +50,41 @@
 import teammates.storage.api.StudentsDb;
 
 public class FeedbackSessionsLogic {
-
+    
+    private static final String ASSUMPTION_FAIL_DELETE_INSTRUCTOR = "Fail to delete instructor respondant for ";
+    private static final String ASSUMPTION_FAIL_RESPONSE_ORIGIN = "Client did not indicate the origin of the response(s)";
+    private static final String ERROR_NUMBER_OF_RESPONSES_EXCEEDS_RANGE = "Number of responses exceeds the limited range";
+    private static final String ERROR_SENDING_EMAILS = "Error while sending emails :";
+    private static final String ERROR_SESSION_UNPUBLISH_ALREADY = "Session is already unpublished.";
+    private static final String ERROR_SESSION_UNPUBLISH_NON_EXISTENT = "Trying to unpublish a non-existent session.";
+    private static final String ERROR_SESSION_UNPUBLISH_PRIVATE = "Private session can't be unpublished.";
+    private static final String ERROR_SESSION_PUBLISH_ALREADY = "Session is already published.";
+    private static final String ERROR_SESSION_PUBLISH_NON_EXISTENT = "Trying to publish a non-existent session.";
+    private static final String ERROR_SESSION_PUBLISH_PRIVATE = "Private session can't be published.";
+    private static final String KEY_FROM_SECTION = "fromSection";
+    private static final String KEY_IN_SECTION = "inSection";
+    private static final String KEY_IS_INCLUDE_RESPONSE_STATUS = "isIncludeResponseStatus";
+    private static final String KEY_QUESTION_ID = "questionId";
+    private static final String KEY_RANGE = "range";
+    private static final String KEY_SECTION = "section";
+    private static final String KEY_TO_SECTION = "toSection";
+    private static final String KEY_VIEW_TYPE = "viewType";
+    private static final String VALUE_FALSE = "false";
+    private static final String VALUE_TRUE = "true";
+    private static final String ERROR_NON_EXISTENT_FS_STRING_FORMAT = "Trying to %s a non-existent feedback session: %s/%s";
+    private static final String ERROR_NON_EXISTENT_TARGET_STRING_FORMAT = "Error getting feedback session(s): %s does not exist.";
+    private static final String TARGET_STUDENT = "Student";
+    private static final String TARGET_COURSE = "Course";
+    private static final String ERROR_NON_EXISTENT_FS_GET_FOR_COURSE = String.format(ERROR_NON_EXISTENT_TARGET_STRING_FORMAT,
+                                                                                     TARGET_COURSE);
+    private static final String ERROR_NON_EXISTENT_FS_GET_FOR_STUDENT = String.format(ERROR_NON_EXISTENT_TARGET_STRING_FORMAT,
+                                                                                      TARGET_STUDENT);
+    private static final String ACTION_GET = "get";
+    private static final String ACTION_UPDATE = "update";
+    private static final String ACTION_CHECK = "check";
+    private static final String ACTION_REMIND = "remind";
+    private static final String ACTION_VIEW = "view";
```

Comment:
I think for each of this, can just use something like:
~~~
private static final String ERROR_GET_NON_EXISTENT_FS =
        String.format(ERROR_NON_EXISTENT_FS_STRING_FORMAT, "get");
~~~

So we can remove all the `ACTION_*`